examples/viewer.js

```diff
@@ -264,3 +373,233 @@ function sendViewerMessage(message) {
+                "<tr><td>Peer Connection: </td><td>" + connectionString + "</td></tr>" +
+                "<tr><td>Avg RTT: </td><td>" +  testAvgRTT.toFixed(3) + " sec</td></tr>" +
+                "<tr><td>Video Resolution: </td><td>" +  videoWidth+ " x " + videoHeight + "</td></tr>" +
+                "<tr><td>Avg Video bitrate: </td><td>" + testAvgVbitrate.toFixed(1) + " kbps</td></tr>" +
+                "<tr><td>Avg Video Frame Rate: </td><td>" + testAvgFPS.toFixed(2) + " FPS</td></tr>" +
+                "<tr><td>Avg Frame Drop : </td><td>" + testAvgDropPer.toFixed(2) + " %</td></tr>" +
+                "<tr><td>Avg Video Jitter: </td><td>" + testAvgVjitter.toFixed(3) + " sec</td></tr>" +
+                "<tr><td>Avg Audio bitrate: </td><td>" + testAvgAbitrate.toFixed(1) + " kbps</td></tr>" +
+                "<tr><td>Avg Audio Jitter: </td><td>" + testAvgAjitter.toFixed(3) + " sec</td></tr></table>";
+            } 
+            // Update the page
+            $('#dqp-test')[0].innerHTML =html_str; 
+
+            // Display ongoing live stats.
+            html_str = "<table><tr><td>VIEWER Start: </td><td>" + viewer_button_pressed.toISOString() + "</td></tr>" +
+            "<tr><td>TRACK Start: </td><td>" + initialDate.toISOString() + "</td></tr>" +
+            "<tr><td>DECODED Start: </td><td>" + statStartDate.toISOString() + "</td></tr>" +
+            "<tr><td>Time Connected: </td><td>" + statRunTime + " sec</td></tr>" +
+            "<tr><td>Peer Connection: </td><td>" + connectionString + "</td></tr>" +
+            "<tr><td>RTT: </td><td>" + rttCurrent.toFixed(3) + " sec</td></tr>" +
+            "<tr><td><u>VIDEO: </u></td></tr>" +
+            "<tr><td></td><td>Resolution: </td><td>" + videoWidth+ " x " + videoHeight + "</td></tr>" +
+            "<tr><td></td><td>Bitrate: </td><td>" + videoBitrate + " kbps</td></tr>" +
+            "<tr><td></td><td>Frame Rate: </td><td>" + videoFramerate + " FPS</td></tr>" +
+            "<tr><td></td><td>Frames Dropped: </td><td>" + curDropPercent.toFixed(2) + " %</td></tr>" +
+            "<tr><td></td><td>Jitter: </td><td>" + videojitter.toFixed(3) + " sec</td></tr>" +
+            "<tr><td><u>AUDIO: </u></td></tr>" +
+            "<tr><td></td><td>Bitrate : </td><td>" + audioBitrate + " kbps</td></tr>" +
+            "<tr><td></td><td>Samples Received: </td><td>" + audioSamplesReceived + "</td></tr>" +
+            "<tr><td></td><td>Jitter: </td><td>" + audiojitter.toFixed(3) + " sec</td></tr></table>";
+            // Update the page
+            $('#webrtc-live-stats')[0].innerHTML =html_str; 
+
+            previousTime = currentTime;
+
+        } else {
+            html_str = "<table><tr><strong><FONT COLOR=RED>WAITING FOR STREAM STATS...</FONT></strong></tr></table>"
+            //Update the page
+            $('#dqp-test')[0].innerHTML =html_str; 
+            console.log("[DQP TEST] Waiting for stream stats...");
+
+            previousTime = currentTime;
+            return;
```

Comment:
minor: return is unnecessary as we are already at the end of the function

Reply:
removed!